src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -102,7 +103,7 @@
      *        DetailAST of Javadoc comment
      * @return DetailNode tree of Javadoc comment
      */
-    public ParseStatus parseJavadocAsDetailNode(DetailAST javadocCommentAst) {
+    public ParseStatus parseJavadoc(DetailAST javadocCommentAst) {
         blockCommentLineNumber = javadocCommentAst.getLineNo();
```

Comment:
This is an evil change. The method name is "parseJavadocAsDetailNode" while return value is "ParseStatus".
I renamed this method to "parseJavadoc" and added a new method named "parseJavadocAsDetailNode" that really returns the DetailNode.
I'm guessing possible problems for plugin developers here.